MAUI/DataForm/grouping.md

```diff
@@ -0,0 +1,308 @@
+---
+layout: post
+title: Grouping in .NET MAUI DataForm control | Syncfusion
+description: Learn here all about the grouping support in Syncfusion .NET MAUI DataForm (SfDataForm) control and more.
```

Comment:
Learn here all about the grouping support in Syncfusion .NET MAUI DataForm (SfDataForm) control.